example/spec/parallel/testAPI/oneTimeRule.js

```diff
@@ -126,22 +127,35 @@ describe('When I create a one-time rule via the Cumulus API', () => {
   });
 
   describe('When listing the rules via the API', () => {
-    let listRulesResponse = '';
+    let listRules = '';
 
     beforeAll(async () => {
-      listRulesResponse = await rulesApiTestUtils.listRules({
+      const listRulesResponse = await rulesApiTestUtils.listRules({
         prefix: config.stackName
       });
 
-      console.log(JSON.stringify(listRulesResponse));
+      listRules = JSON.parse(listRulesResponse.body);
+      console.log(JSON.stringify(listRules));
     });
 
     it('the rule is returned with the listed rules', () => {
-      const rule = listRulesResponse.results.find((result) => result.name === helloWorldRule.name);
+      const rule = listRules.results.find((result) => result.name === helloWorldRule.name);
       expect(rule).toBeDefined();
 
       const ruleCopy = removeRuleAddedParams(rule);
       expect(ruleCopy).toEqual(helloWorldRule);
     });
   });
+
+  it('the rule can be updated', async () => {
+    const updatingRuleResponse = await rulesApiTestUtils.updateRule({
```

Comment:
I may be misunderstanding the AC, but I think we would want the update to the rule to happen before the ingest so that we can test that the configured rule for the task has whatever updates you made